### DISCUSSION TITLE
Getting an error when installing via vim plug - x Post-update hook for vim-clap ... Vim(call):E117: Unknown function: clap#job#daemon#start

### DISCUSSION THREAD

**Discussion by keithlxs** [liuchengxu/vim-clap#1027] [General]
Hello

I'm getting an error when I'm installing vim-clap in NeoVim (0.9.4). I use vim plug and I get this error;

x Post-update hook for vim-clap ... Vim(call):E117: Unknown function: clap#job#daemon#start

I'm using the following command in my config

Plug('liuchengxu/vim-clap', {
['do'] = function()
vim.call(':Clap install-binary!')
end
})

Apparently there seems to be an issue with the clap.vim file in the plugins folder. I'm not sure what else I could be doing wrong or if I'm missing something else. Any assistance would be greatly appreciated.

**Comment by liuchengxu** [comment]
This minimal vimrc works fine on my machine. 

```vim
silent! if plug#begin('/tmp/plugged')
    Plug 'liuchengxu/vim-clap', { 'do': ':Clap install-binary!' }
    call plug#end()
endif
```

In addition, the latest commit https://github.com/liuchengxu/vim-clap/commit/5a155263d238d30bd1c6e4101681f4f5737a21b8 brings some refinements to the install experience, please pull the latest commit and give it another spin. If it still does not work somehow, try running `:Clap install-binary!` manually then.
